experimental/relativetime/src/provider.rs

```diff
@@ -0,0 +1,124 @@
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+// Provider structs must be stable
+#![allow(clippy::exhaustive_structs, clippy::exhaustive_enums)]
+
+//! Data provider struct definitions for this ICU4X component.
+//!
+//! Read more about data providers: [`icu_provider`]
+
+use alloc::borrow::Cow;
+use icu_provider::{yoke, zerofrom, DataError};
+use zerovec::ZeroMap;
+
+/// Relative time pattern V1 data struct.
+#[icu_provider::data_struct(
+    LongSecondRelativeTimeFormatDataV1Marker = "relativetime/long/second@1",
+    ShortSecondRelativeTimeFormatDataV1Marker = "relativetime/short/second@1",
+    NarrowSecondRelativeTimeFormatDataV1Marker = "relativetime/narrow/second@1",
+    LongMinuteRelativeTimeFormatDataV1Marker = "relativetime/long/minute@1",
+    ShortMinuteRelativeTimeFormatDataV1Marker = "relativetime/short/minute@1",
+    NarrowMinuteRelativeTimeFormatDataV1Marker = "relativetime/narrow/minute@1",
+    LongHourRelativeTimeFormatDataV1Marker = "relativetime/long/hour@1",
+    ShortHourRelativeTimeFormatDataV1Marker = "relativetime/short/hour@1",
+    NarrowHourRelativeTimeFormatDataV1Marker = "relativetime/narrow/hour@1",
+    LongDayRelativeTimeFormatDataV1Marker = "relativetime/long/day@1",
+    ShortDayRelativeTimeFormatDataV1Marker = "relativetime/short/day@1",
+    NarrowDayRelativeTimeFormatDataV1Marker = "relativetime/narrow/day@1",
+    LongWeekRelativeTimeFormatDataV1Marker = "relativetime/long/week@1",
+    ShortWeekRelativeTimeFormatDataV1Marker = "relativetime/short/week@1",
+    NarrowWeekRelativeTimeFormatDataV1Marker = "relativetime/narrow/week@1",
+    LongMonthRelativeTimeFormatDataV1Marker = "relativetime/long/month@1",
+    ShortMonthRelativeTimeFormatDataV1Marker = "relativetime/short/month@1",
+    NarrowMonthRelativeTimeFormatDataV1Marker = "relativetime/narrow/month@1",
+    LongQuarterRelativeTimeFormatDataV1Marker = "relativetime/long/quarter@1",
+    ShortQuarterRelativeTimeFormatDataV1Marker = "relativetime/short/quarter@1",
+    NarrowQuarterRelativeTimeFormatDataV1Marker = "relativetime/narrow/quarter@1",
+    LongYearRelativeTimeFormatDataV1Marker = "relativetime/long/year@1",
+    ShortYearRelativeTimeFormatDataV1Marker = "relativetime/short/year@1",
+    NarrowYearRelativeTimeFormatDataV1Marker = "relativetime/narrow/year@1"
+)]
+#[derive(Debug, Clone, Default, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize))]
+#[cfg_attr(
+    feature = "datagen", 
+    derive(serde::Serialize, databake::Bake),
+    databake(path = icu_relativetime::provider)
+)]
+#[yoke(prove_covariance_manually)]
+pub struct RelativeTimePatternDataV1<'data> {
+    #[cfg_attr(feature = "serde", serde(borrow))]
+    /// The display name of the relative time format pattern.
+    /// Empty if display name is not present in CLDR data.
+    pub display_name: Cow<'data, str>,
```

Comment:
Nit: Do we need the display name for RelativeTimeFormatter? We need it for Intl.DisplayNames, but that should go into a separate key in the future.

Reply:
Removed "displayName" from V1Data struct.